oeps/oep-0027-course-metadata-authoring.rst

```diff
@@ -0,0 +1,138 @@
+===================================
+OEP-0027: Course Metadata Authoring
+===================================
+
++-----------------+----------------------------------------------------------------+
+| OEP             | :doc:`OEP-0027 </oeps/oep-0027-arch-course-metadata-authoring>`|
++-----------------+----------------------------------------------------------------+
+| Title           | Course Metadata Authoring                                      |
++-----------------+----------------------------------------------------------------+
+| Last Modified   | 2018-07-13                                                     |
++-----------------+----------------------------------------------------------------+
+| Authors         | Farhanah Sheets, Michael Roytman, Nimisha Asthagiri            |
++-----------------+----------------------------------------------------------------+
+| Arbiter         | <TODO>                                                         |
++-----------------+----------------------------------------------------------------+
+| Status          | Under Review                                                   |
++-----------------+----------------------------------------------------------------+
+| Type            | Architecture                                                   |
++-----------------+----------------------------------------------------------------+
+| Created         | 2018-07-13                                                     |
++-----------------+----------------------------------------------------------------+
+| `Review Period` | <TODO>                                                         |
++-----------------+----------------------------------------------------------------+
+
+Context
+-------
+
+In the Open edX system, course content authoring occurs in the Studio tool, which focuses mainly on authoring of
+course structure and content within a **Course Run**. A Course Run is discoverable on the edX marketing site via
+the context of a **Catalog Course**. The **Catalog Course** data authoring needs a good user experience.
```

Comment:
Catalog Course?  What does this mean?  In the catalog service (course-discovery repo) we have the concept of course and course run.  The course is the unchanging metadata around a course (i.e. CS50) and the course run is the instance of that course that is running during a particular period of time.  I think naming is important and Catalog Course doesn't have much meaning to me as a frequent consumer of the course-discovery service.